src/utils/FMatrix4x3.h

```diff
@@ -0,0 +1,63 @@
+//Copyright (c) 2020 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#ifndef FMATRIX4X3_H
+#define FMATRIX4X3_H
+
+namespace cura
+{
+
+class FPoint3;
+class Point3;
+class Ratio;
+
+/*!
+ * A 4x3 affine transformation matrix.
+ *
+ * This matrix behaves as if it's a 4x4 transformation matrix, but the bottom
+ * row is always identity.
+ */
+class FMatrix4x3
+{
+public:
+	/*!
```

Comment:
This is suddenly indented with tabs. The other code of CuraEngine seems to use spaces.

Reply:
Fixed all of them. I need to take another hard look at my IDE because this keeps going wrong. Even restarting the IDE sometimes fixes it so it's definitely buggy...